CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unversioned
 
+- Major: Added the ability to autocomplete Fossabot & Streamelements commands. (#3209)
```

Comment:
```suggestion
- Major: Added support for bot command autocompletion using Chatterino's command list API: https://github.com/Chatterino/command-list-api (#3209)
```

Reply:
I don't think that an average Chatterino user would care about the github link in a changelog entry. If someone's interested more into how this feature is going to work they have a PR number and can read up detail on the PR.

Reply:
```suggestion
- Major: Added support for bot command autocompletion using Chatterino's command list API. (#3209)
```